TopNews/app/src/main/java/com/fisfam/topnews/pojo/Category.java

```diff
@@ -16,4 +20,16 @@ public int getIconResource() {
     public String getCategoryName() {
         return mCategoryName;
     }
+
+    public static ArrayList<Category> getCategoryList() {
```

Comment:
đã bảo không để cái này trong class Category, add trực tiếp nó từ Home là được rồi.

Reply:
De the nay ko tot hon a. Tsao lai cu phai vut vao Home

Reply:
nếu thích cái List này thì không hơi đâu mỗi lần get lại tạo List mới cả.
Tạo 1 static final List trong class thôi..